kf_lib_data_ingest/network/utils.py

```diff
@@ -0,0 +1,114 @@
+"""
+Common network (HTTP, TCP, whatever) related functionality
+"""
+import cgi
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def get_file(url, dest_obj, **kwargs):
```

Comment:
This could be more generic - return either regular resource or file like obj but I left it file specific for now.